packages/dev/core/src/Cameras/arcRotateCamera.ts

```diff
@@ -827,6 +829,33 @@ export class ArcRotateCamera extends TargetCamera {
         return true;
     }
 
+    /**
+     * Interpolates the camera to a goal state.
+     * @param alpha Defines the goal alpha.
+     * @param beta Defines the goal beta.
+     * @param radius Defines the goal radius.
+     * @param target Defines the goal target.
+     * @param targetScreenOffset Defines the goal target screen offset.
+     */
+    public interpolateTo(alpha = this.alpha, beta = this.beta, radius = this.radius, target = this.target, targetScreenOffset = this.targetScreenOffset): void {

```

Comment:
This is a new public API, so please speak up if you have any concerns!

Reply:
this interpolation will trigger a change in the goal properties, but there is no way to know when the goal has been reached. Thinking about future use cases, it feels like there are a few that will require (for example) an `onReached` callback to the interpolate function.

Reply:
We could accept callbacks, or we could return a promise. We could also add that in the future when needed since it is a non-breaking change.

Reply:
Returning a promise will feel very awkward here (at least to me). I agree that it can be added in the future

Reply:
We agreed we can do this as an improvement in the future if/when needed as it shouldn't be a breaking change.